docs/userguide.md

```diff
@@ -211,3 +211,49 @@ with httpcore.ConnectionPool() as pool:
         cache_pool.get("https://example.com/cachable-endpoint")
         response = cache_pool.get("https://example.com/cachable-endpoint") # from the cache
 ```
+
+### Temporarily Disabling the Cache
+
```

Comment:
I believe this section could be removed. if we want to include references to extensions, headers, or other sections, we will do so in a single pull request as part of a documentation refactoring.

Reply:
On my to-do list is some proposed revisions to the docs. We can do that then!